packages/primitives/stepper/src/stepper-root-context.token.ts

```diff
@@ -0,0 +1,15 @@
+import { inject, InjectionToken } from '@angular/core';
+
+export interface StepperRootContext {
+    value: any;
+    orientation: any;
+    dir: any;
+    linear: any;
+    totalStepperItems: any;
+}
+
+export const STEPPER_ROOT_CONTEXT = new InjectionToken<StepperRootContext>('StepperRootContext');
+
+export function injectStepperRootContext(): StepperRootContext {
```

Comment:
should we specify that context is optional?

```suggestion
export function injectStepperRootContext(): StepperRootContext | undefined {
    return inject(STEPPER_ROOT_CONTEXT, { optional: true });
```

Reply:
Why do we need this?

Reply:
nvm, I thought it was optional